packages/bitcore-wallet-service/src/lib/chain/btc/index.ts

```diff
@@ -210,6 +210,19 @@ export class BtcChain implements IChain {
     }
   }
 
+  checkScriptOutput(output) {
+    if (output.script) {
+      if (typeof output.script !== 'string') {
+        return Errors.SCRIPT_TYPE;
+      }
+
+      // check OP_RETURN
+      if (output.script.startsWith('6a') && output.amount != 0) {
+        return Errors.SCRIPT_OP_RETURN_AMOUNT;
+      }
+    }
+  }
```

Comment:
This is blank check for a non OP_RETURN script. Perhaps to protect against potential script attacks, we should _only_ support OP_RETURN scripts for now.

Reply:
Done. I added a check to ensure that script is only of type OP_RETURN